src/components/TransactionsList/TransactionsList.js

```diff
@@ -1,21 +1,38 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import './TransactionsList.css';
+
 import Transaction from '../Transaction/Transaction';
 
 const TransactionsList = ({ transactions }) => (
 	<div className="TransactionsList">
-		{ transactions.map(transaction => (
-			<Transaction
-				key={transaction.transactionHash}
-				{...transaction}
-			/>
-		)) }
+		{transactions.length > 0 &&
+			transactions.map(transaction => (
+				<Transaction
+					key={transaction.transactionHash}
+					{...transaction}
+				/>
+			))
+		}
+		{transactions.length <= 0 &&
+			<p className="no-results">
+		]		No transactions found!
+				<span className="icon">😖</span>
```

Comment:
Emoji in the code!?!?!?

Reply:
Please see this site for emoji support on relevant browsers - http://caniemoji.com/